core/src/main/java/nl/nn/adapterframework/validation/JavaxXmlValidator.java

```diff
@@ -77,11 +77,11 @@ public void configure(String logPrefix) throws ConfigurationException {
 		}
 		super.configure(logPrefix);
 	}
-	
+
 	@Override
-	protected void init() throws ConfigurationException {
-		if (needsInit) {
-			super.init();
+	public void start() throws ConfigurationException {
+		if(!isStarted()) {
+			super.start();
```

Comment:
Zo krijg je natuurlijk nooit die ConfigurationException("cannot start an already started validator");

Reply:
Dat klopt, het was of dit of 56 junit tests aanpassen...